keps/sig-release/3744-stay-on-supported-go-versions/README.md

```diff
@@ -0,0 +1,445 @@
+
+n/a
+
+###### Are there any tests for feature enablement/disablement?
+
+n/a
+
+### Rollout, Upgrade and Rollback Planning
+
+###### How can a rollout or rollback fail? Can it impact already running workloads?
+
+n/a
+
+###### What specific metrics should inform a rollback?
+
+n/a
+
+###### Were upgrade and rollback tested? Was the upgrade->downgrade->upgrade path tested?
+
+n/a
+
+###### Is the rollout accompanied by any deprecations and/or removals of features, APIs, fields of API types, flags, etc.?
+
+No
+
+### Monitoring Requirements
+
+n/a
+
+###### How can an operator determine if the feature is in use by workloads?
+
+n/a
+
+###### How can someone using this feature know that it is working for their instance?
+
+n/a
+
+###### What are the reasonable SLOs (Service Level Objectives) for the enhancement?
+
+n/a
+
+###### What are the SLIs (Service Level Indicators) an operator can use to determine the health of the service?
+
+n/a
+
+###### Are there any missing metrics that would be useful to have to improve observability of this feature?
+
+n/a
+
+### Dependencies
+
+###### Does this feature depend on any specific services running in the cluster?
+
+No
+
+### Scalability
+
+###### Will enabling / using this feature result in any new API calls?
+
+No
+
+###### Will enabling / using this feature result in introducing new API types?
+
+No
+
+###### Will enabling / using this feature result in any new calls to the cloud provider?
+
+No
+
+###### Will enabling / using this feature result in increasing size or count of the existing API objects?
+
+No
+
+###### Will enabling / using this feature result in increasing time taken by any operations covered by existing SLIs/SLOs?
+
+No
+
+###### Will enabling / using this feature result in non-negligible increase of resource usage (CPU, RAM, disk, IO, ...) in any components?
+
+No
+
+### Troubleshooting
+
+###### How does this feature react if the API server and/or etcd is unavailable?
+
+n/a
+
+###### What are other known failure modes?
+
+n/a
+
+###### What steps should be taken if SLOs are not being met to determine the problem?
+
+n/a
+
+## Implementation History
+
+<!--
+Major milestones in the lifecycle of a KEP should be tracked in this section.
+Major milestones might include:
+- the `Summary` and `Motivation` sections being merged, signaling SIG acceptance
+- the `Proposal` section being merged, signaling agreement on a proposed design
+- the date implementation started
+- the first Kubernetes release where an initial version of the KEP was available
+- the version of Kubernetes where the KEP graduated to general availability
+- when the KEP was retired or superseded
+-->
+
+- 2023-01-17: Created
+
+## Drawbacks
+
+* The primary drawback is risk of a user-facing regression in a patch release due to a go version change.
+  Most of this KEP is focused on ways to detect and guard against that risk.
+* This also creates some additional work for maintainers to track changes 
```

Comment:
I hope that we do not create a queue of issues to be done after the golang update has been merged while waiting 3 months for adoption, but I see no real way around that.

Reply:
Irrelevant per https://github.com/kubernetes/enhancements/pull/3749#discussion_r1085430246